modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -415,10 +415,15 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
       throw new Error('deprecated');
     }
 
-    const formats = param && param.anyFormat ? undefined : ['default' as const];
```

Comment:
So previously the `anyFormat` flag controlled this behavior. What you want to do here is make `anyFormat = true` the default.